--- handlers/aiortc_handler.py	2024-09-17 11:53:36.114144000 -0500
+++ handlers/aiortc_handler-m.py	2024-09-16 16:17:23.031630000 -0500
@@ -7,6 +7,7 @@
     RTCSessionDescription,
     RTCRtpTransceiver,
     MediaStreamTrack,
+    RTCConfiguration
 )
 import sdp_transform
 from .sdp import common_utils
@@ -165,7 +166,10 @@
                 "video", options.extendedRtpCapabilities
             ),
         }
-        self._pc = RTCPeerConnection()
+
+        configuration = RTCConfiguration(iceServers=iceServers)
+           
+        self._pc = RTCPeerConnection(configuration=configuration)
 
         @self._pc.on("iceconnectionstatechange")
         def on_iceconnectionstatechange():
---  handlers/handler_interface.py	2024-09-17 11:53:45.337137000 -0500
+++  handlers/handler_interface-m.py	2024-09-16 16:15:00.478833000 -0500
@@ -52,7 +52,7 @@
         dtlsParameters: DtlsParameters,
         extendedRtpCapabilities: ExtendedRtpCapabilities,
         sctpParameters: Optional[SctpParameters] = None,
-        iceServers: Optional[RTCIceServer] = None,
+        iceServers: Optional[List[RTCIceServer]] = None,
         iceTransportPolicy: Optional[Literal["all", "relay"]] = None,
         additionalSettings: Optional[Any] = None,
         proprietaryConstraints: Optional[Any] = None,